### ISSUE TITLE
Vue 3 Complaining createVueWait() is not a function

### ISSUE THREAD

**Issue by Mohanrau** [f/vue-wait#107]
@vue/cli 4.5.13

![error](https://user-images.githubusercontent.com/1044660/127433733-632b1672-7ed5-444f-93d6-c249099a3c42.PNG)
![error-1](https://user-images.githubusercontent.com/1044660/127433735-bb538ffb-90ae-4e3b-a57a-020799d861b2.PNG)

**Comment by VadymGidulian** [comment]
This helped me to get around the issue:
```js
// webpack.config.js
resolve: {
	alias: {
		'vue-wait$': 'vue-wait/src/vue-wait.js'
	}
}
```

**Comment by f** [comment]
This should be fixed with the version v1.5.1

**Comment by VadymGidulian** [comment]
@f Thanks! But I got this error on 1.5.1:
```
TS2345: Argument of type 'Object' is not assignable to parameter of type 'Plugin_2'.
  The 'Object' type is assignable to very few other types. Did you mean to use the 'any' type instead?
    The 'Object' type is assignable to very few other types. Did you mean to use the 'any' type instead?
      Property 'install' is missing in type 'Object' but required in type '{ install: PluginInstallFunction; }'.
```

Workaround:
```ts
import {Plugin, createApp} from 'vue';
import {createVueWait}     from 'vue-wait'

const app = createApp(Index)
	.use(createVueWait({...}) as Plugin)
	.mount('#app');
```
